xml/templates/message_templates/event_online_receipt_html.tpl

```diff
@@ -244,6 +244,22 @@
                 {/if}
               {/foreach}
             {/if}
+            {if !$isShowLineItems}
+              {foreach from=$participants key=index item=currentParticipant}
+                {if $isPrimary || {participant.id} === $currentParticipant.id}
+                  {foreach from=$currentParticipant.line_items key=index item=currentLineItem}
+                  <tr>
+                    <td {$valueStyle}>
+                      {$currentLineItem.label} {if $isPrimary} - {$currentParticipant.contact.display_name}{/if}
+                    </td>
+                    <td {$valueStyle}>
+                      {$currentLineItem.line_total|crmMoney:$currency}
+                    </td>
+                  </tr>
+                  {/foreach}
+                {/if}
+              {/foreach}
+            {/if}
             {if $isShowTax && {contribution.tax_amount|boolean}}
```

Comment:
Not a blocker at all, but this condition and the one at 283 are the same, so could remove 282 and 283 to simplify.

Reply:
thanks @larssandergreen - I've pushed a fix